e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -2407,15 +2552,6 @@ const onNextAnchorClick = callback => {
   });
 };
 
-/**
- * Duplicated from metabase/lib/card because Cypress can't handle import from there.
- *
- * @param {string} value
- * @returns object
- */
-const deserializeCardFromUrl = serialized =>
```

Comment:
This function was used to test implementation details which is not what we should be testing.

3 tests were using this function. They started to fail because the `card.dataset_query.query` will now come with an extra empty stage that is useful for filtering (see `Lib.ensureFilterStage`).